packages/components/src/text-control-with-affixes/index.js

```diff
@@ -40,9 +41,15 @@ class TextControlWithAffixes extends Component {
 			describedby.push( `${ id }__suffix` );
 		}
 
+		const affixesClasses = classnames( 'text-control-with-affixes', {
+			'text-control-with-prefix': prefix,
+			'text-control-with-suffix': suffix,
+			disabled: this.props.disabled,
```

Comment:
Looks like this is missing from the `propTypes` declaration.

Reply:
Fixed in 022bc34.